pxtsim/runtime.ts

```diff
@@ -34,16 +34,14 @@ namespace pxsim {
         }
 
         export function now(): number {
-            return Date.now();
```

Comment:
Makes sense. But is it okay if we override startTime's value with perfNow instead or is something else using it?